packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +84,27 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+/// An expansion panel that allows for radio-like functionality.
+///
+/// A unique identifier [value] must be assigned to each panel.
+class ExpansionPanelRadio extends ExpansionPanel {
+
+  /// A subclass of expansion panel that allows for radio functionality.
+  ///
+  /// A unique identifier [value] must be passed into the constructor. The
+  /// [headerBuilder], [body], [value] must not be null.
+  ExpansionPanelRadio({
+    @required this.value,
+    @required ExpansionPanelHeaderBuilder headerBuilder,
+    @required Widget body,
+  }) : assert(value != null),
+       super(body: body, headerBuilder: headerBuilder);
+
+  /// The value that uniquely identifies a radio panel.
```

Comment:
Add why it exists: "so that the currently selected radio panel may be identified."